meta-balena-zeus/recipes-kernel/linux/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,17 @@
+inherit balena-linux-firmware
```

Comment:
Can this be moved to meta-balena-common? Even though the older versions may not be IWLWIFI_FW_TOCLEAN aware they should not error out if this moved to the common area, right?

Reply:
@floion This is an interesting case.
`IWLWIFI_FW` is affected via poky/kernel both.
The poky recipe adds all the firmware but it takes a ton of space which we'd like to clean up.
But knowing which wifi firmware to clean up is not a simple decision. The Kernel iwlwifi driver version specifies what version it can support. The kernel is part of the BSP layer.

Specifying which firmware to clean should technically be BSP level depending on the kernel version is running.

I would consider these as sensible 'defaults'. (especially now that we have them in meta-balena already).

But future changes to `IWLWIFI_FW_TOCLEAN` and `IWLWIFI_FW_MIN_API` should be at BSP/Integration layer level.

Regarding the specific question about whether this should be in meta-balena-common. Hmm.

I'm not sure. e.g. edison runs sumo and kernel v4.16. Does the iwlwifi driver in that kernel support this minimum wifi firmware? I don't know.

e.g. We could ship only the latest firmware via meta-balena-common. But then older devices with older kernels might not support latest firmware.

As I said. This is technically a BSP/Integration layer decision depending on the kernel running.

I think we have basically gotten away with it until now.

To sum up a simple answer to your question. This lives in meta-balena-zeus layer.